lte/gateway/python/magma/pipelined/main.py

```diff
@@ -76,6 +80,13 @@ def main():
     if 'virtual_mac' not in service.config:
         service.config['virtual_mac'] = get_if_hwaddr(service.config.get('bridge_name'))
 
+    service.config['uplink_port'] = OFPP_LOCAL
```

Comment:
why this overriding of the service configs? ideally the YAML files should be owning the configuration, not the other way around? can this be done using another variable?

Reply:
uplink_port is not part of config.   it is derived from ovs_uplink_port_name.
This is not new code, it is moved from inout controller to main, since two controller are going to use same config.
As you can see in the file plan is to move all config parsing in separate module from individual controllers. First step is to move to main.py file.